starter-code/main.js

```diff
@@ -0,0 +1,215 @@
+									if (error) {
+									console.log(error);
+									rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+									} else {
+									console.log(result);
+									rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+									}
+									})
+							break;
+							case "6":
+								db.collection('companies').find({"offices.city": "Barcelona"},{name: 1, city:1, _id: 0}).toArray((error, result) => {
+										if (error) {
+										console.log(error);
+										rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+										} else {
+										console.log(result);
+										rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+										}
+										})
+								break;
+								case "7":
+									db.collection('companies').find({},{name: 1, number_of_employees:1, _id: 0}).limit(10).sort({ number_of_employees: -1 }).toArray((error, result) => {
+											if (error) {
+											console.log(error);
+											rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+											} else {
+											console.log(result);
+											rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+											}
+											})
+									break;
+									case "8":
+										db.collection('companies').find({"name": "Facebook"},{_id: 0}).toArray((error, result) => {
+												if (error) {
+												console.log(error);
+												rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+												} else {
+												console.log(result);
+												rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+												}
+												})
+										break;
+										case "9":
+											db.collection('companies').find({"name": "Facebook"},{name: 1, number_of_employees:1, _id: 0}).toArray((error, result) => {
+													if (error) {
+													console.log(error);
+													rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+													} else {
+													console.log(result);
+													rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+													}
+													})
+											break;
+											case "10":
+												db.collection('companies').find({"name": "Facebook"},{name: 1, products:1, _id: 0}).toArray((error, result) => {
+														if (error) {
+														console.log(error);
+														rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+														} else {
+														console.log(result);
+														rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+														}
+														})
+												break;
+												case "11":
+													db.collection('companies').find({"relationships.is_past" : "false"},{name: 1, "relationships.person.first_name": 1, "relationships.person.last_name": 1, _id: 0}).toArray((error, result) => {
```

Comment:
Muy buena clasificación desde la misma query, te has ahorrado hacer el procedimiento desde el código!

Tal que así por ejemplo:

              } else {
                console.log(result[0].relationships.filter(e => {
                  if (e.is_past == false) return e;
                }).map(e => {
                  return `${e.person.first_name} ${e.person.last_name}`;
                }));
                rl.question(`\nType enter to continue: `, (answer) => {
                  mainMenu()
                });
              }